### PR TITLE
Umi - Added Metaplex Plugins Overview

### PR DESCRIPTION
**Pull request by tonyboylehub**
Users where getting confused on "where to go next" during their Umi setup process so I've added a Metaplex Umi plugins description page that should benefit new users with an overview of each plugin and also outlined their reprospective jobs/tasks each plugin/program performs to give them a form of direction in which to explore next.

Link is now the 4th option of Introduction in the Umi Menu after 'Connecting to Umi'.

### CODE REVIEW

File: src/pages/umi/metaplex-umi-plugins.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+title: Meteplex Umi Plugins
+metaTitle: Umi - Meteplex Umi Plugins
+description: An overview of Meteplex Umi Plugins.
+---
+
+Metaplex programs have been generated via Kinobi to work and run as plugin's via Umi. Each program in the Metaplex library has a different use and purpose in the Solana eco system.
```
Comment (MarkSackerberg) [inline src/pages/umi/metaplex-umi-plugins.md:7]:
```suggestion
Metaplex programs have been generated via Kinobi to work and run as plugin's via Umi. Each program in the Metaplex library has a different use and purpose in the Solana eco system. You can find even more Plugins to use with Umi in the [interface implementations page](/umi/implementations)!
```

**Review by MarkSackerberg — COMMENTED** [review]
Some nitpicks. Let’s discuss them before merging.

File: src/pages/umi/metaplex-umi-plugins.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+- Data Fetching
+- Minting
+- Updating
+- Transfering
+- Burning
+- Delegating
+- Collection Management
+
+
+## [Toolbox](/toolbox)
```
[the remark this answers is not in the document]
Comment (tonyboylehub) [reply src/pages/umi/metaplex-umi-plugins.md:62]: I'm happy with it as toolbox but impartial, I think it explains things ok in the description. I'm

File: src/pages/umi/metaplex-umi-plugins.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+- Minting NFTs
+- Selling NFTs.
+
+## [DAS API](/das-api)
+
+The state data of uncompressed NFTs is all stored in on-chain accounts. This is expensive at scale. Compressed NFTs save space by encoding the state data into an on-chain Merkle tree. The detailed account data is not stored on-chain, but in data stores managed by RPC providers. The Metaplex Digital Asset Standard (DAS) API represents a unified interface for interacting with digital assets on Solana, supporting both standard (Token Metadata) and compressed (Bubblegum) assets.
+
+Program feature set includes:
+
+- Data fetching
```
Comment (MarkSackerberg) [inline src/pages/umi/metaplex-umi-plugins.md:37]:
```suggestion
- Fast Data fetching, including compressed NFTs
```
```suggestion
- Data fetching
```

**Review by MarkSackerberg — APPROVED** [review]

**Pull request merged**